README.md

```diff
@@ -68,89 +120,36 @@ Following must be available locally:
         * ```.\venv\scripts\activate```
         * ```.\devops\win\install.bat```
 
-### Environment vars:
-As the project is open source, we have to keep secrets out of the repo.  You will need to setup the following env vars locally:
-
-* **TM_DB** - This is for the PostGIS connection string.  If you can't access an existing DB refer to DevOps page to [set up a local DB in Docker](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops#creating-a-local-postgis-database-with-docker)
-* **TM_SECRET** - This is secret key for the TM app used by itsdangerous and flask-oauthlib for entropy
-* **TM_CONSUMER_KEY** - This is the OAUTH Consumer Key used for authenticating the Tasking Manager App in OSM
-* **TM_CONSUMER_SECRET** - This is the OAUTH Consumer Secret used for authenticating the Tasking Manager App in OSM
-* **TM_SMTP_HOST** - The hostname for the SMTP server that is used to send email alerts
-* **TM_SMTP_PORT** - The port number for the SMTP server that is used to send email alerts
-* **TM_SMTP_USER** - The user for the SMTP server that is used to send email alerts
-* **TM_SMTP_PASSWORD** - The password for the SMTP server that is used to send email alerts
-
-* Linux/Mac:
-    * (It is strongly recommended to set these within your .bash_profile so they are available to all processes)
-    * ```export TM_DB=postgresql://USER:PASSWORD@HOST/DATABASE```
-    * ```export TM_SECRET=secret-key-here```
-    * ```export TM_CONSUMER_KEY=oauth-consumer-key-goes-here```
-    * ```export TM_CONSUMER_SECRET=oauth-consumer-secret-key-goes-here```
-    * ```export TM_SMTP_HOST=smtp-server-host-here```
-    * ```export TM_SMTP_PORT=smtp-server-port-here```
-    * ```export TM_SMTP_USER=smtp-server-user-here```
-    * ```export TM_SMTP_PASSWORD=smtp-server-password-here```
-* Windows:
-    * ```setx TM_DB "postgresql://USER:PASSWORD@HOST/DATABASE"```
-    * ```setx TM_SECRET "secret-key-here"```
-    * ```setx TM_CONSUMER_KEY "oauth-consumer-key-goes-here"```
-    * ```setx TM_CONSUMER_SECRET "oauth-consumer-secret-key-goes-here"```
-    * ```setx TM_SMTP_HOST "smtp-server-host-here"```
-    * ```setx TM_SMTP_PORT "smtp-server-port-here"```
-    * ```setx TM_SMTP_USER "smtp-server-user-here"```
-    * ```setx TM_SMTP_PASSWORD "smtp-server-password-here"```
-
-### Creating the DB
-We use [Flask-Migrate](https://flask-migrate.readthedocs.io/en/latest/) to create the database from the migrations directory.  If you can't access an existing DB refer to DevOps page to [set up a local DB in Docker](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops#creating-a-local-postgis-database-with-docker) create the database as follows:
+**Tests**
+The project includes a suite of Unit and Integration tests that you should run after any changes
 
 ```
-python manage.py db upgrade
+python -m unittest discover tests/server
 ```
 
-### Running Locally
+## Create the Database from existing data
```

Comment:
I would change this to `## Create a Fresh Database`. Then if you could add another section immediately following:

```
## Migrating from TM2

We also have provided [a script](devops/tm2-pg-migration/migrationscripts.sql) to migrate your data from the prior tasking manager version (v2) to the current one. Please see [this wiki page](https://github.com/hotosm/tasking-manager/wiki/Migrating-from-TM2-to-TM3) for important information about this process.
```